include/sam_webgui/roswasm_sam.h

```diff
@@ -2,32 +2,74 @@
 #define ROSWASM_SAM_H
 
 #include <roswasm_webgui/roswasm_widget.h>
+#include <roswasm_webgui/imgui/imgui.h>
+#include <roswasm_webgui/imgui/imgui_internal.h>
+
+#include <rosgraph_msgs/Log.h>
+
+#include <std_msgs/String.h>
+#include <std_srvs/SetBool.h>
+#include <diagnostic_msgs/DiagnosticArray.h>
 
 #include <sensor_msgs/NavSatFix.h>
 #include <sensor_msgs/BatteryState.h>
+#include <sensor_msgs/Temperature.h>
+#include <sensor_msgs/FluidPressure.h>
 #include <nav_msgs/Odometry.h>
 
 #include <sam_msgs/PercentStamped.h>
-//#include <sam_msgs/ThrusterRPMs.h>
+
 #include <sam_msgs/ThrusterAngles.h>
 #include <sam_msgs/BallastAngles.h>
 //#include <sam_msgs/Leak.h>
 #include <smarc_msgs/Leak.h>
+
+#include <sam_msgs/ConsumedChargeFeedback.h>
+#include <sam_msgs/CircuitStatusStampedArray.h>
+#include <sam_msgs/ConsumedChargeArray.h>
+#include <sam_msgs/UavcanUpdateBattery.h>
+
+#include <smarc_msgs/DualThrusterFeedback.h>
+#include <smarc_msgs/DualThrusterRPM.h>
 #include <smarc_msgs/ThrusterRPM.h>
+#include <smarc_msgs/CTD.h>
+#include <smarc_msgs/Sidescan.h>
+#include <smarc_msgs/SensorStatus.h>
+
+#include <uavcan_ros_msgs/CircuitStatus.h>
+#include <uavcan_ros_msgs/UavcanNodeStatusNamedArray.h>
+#include <uavcan_ros_msgs/UavcanRestartNode.h>
+
+#include <cola2_msgs/DVL.h>
+
+#include <sbg_driver/SbgEkfEuler.h>
+
+#include <list>
+#include <chrono>
+#include <unordered_map>
+
 
 namespace roswasm_webgui {
 
 bool draw_ballast_angles(sam_msgs::BallastAngles& msg, roswasm::Publisher& pub);
 bool draw_percent(sam_msgs::PercentStamped& msg, roswasm::Publisher& pub);
-//bool draw_thruster_rpms(sam_msgs::ThrusterRPMs& msg, roswasm::Publisher& pub);
+bool draw_thruster_rpms(smarc_msgs::DualThrusterRPM& msg, roswasm::Publisher& pub);
 bool draw_thruster_rpm(smarc_msgs::ThrusterRPM& msg, roswasm::Publisher& pub);
 bool draw_thruster_angles(sam_msgs::ThrusterAngles& msg, roswasm::Publisher& pub);
+// // bool draw_bar_signed(sam_msgs::ThrusterAngles& msg, roswasm::Publisher& pub);
+// bool draw_bar_signed(const ImVec2 pos, const ImVec2 size, const int v_fb, const int rangeBar, const int v_cmd, bool draw_value = true);
+
+extern bool guiDebug;
+extern int drawTabs(int _guiMode, const std::map<int, const char*> _modeMap);
 
 class SamActuatorWidget {
 private:
     TopicWidget<sam_msgs::ThrusterAngles>* thruster_angles;
     TopicWidget<smarc_msgs::ThrusterRPM>* thruster1_rpm;
     TopicWidget<smarc_msgs::ThrusterRPM>* thruster2_rpm;
+
+    TopicWidget<smarc_msgs::DualThrusterRPM>* thruster_rpms;
+    roswasm::Publisher rpm_pub;
```

Comment:
Outdated?

Reply:
@Jollerprutt Should these still be here?

Reply:
no, removed